views/update/update-check-your-answers.html

```diff
@@ -15,8 +57,77 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
 
+      {% if isRemoveJourney %}
+        <span class="govuk-caption-xl govuk-!-padding-bottom-1">
+          {% if appData.entity_name %}
+              {{ appData.entity_name }}
```

Comment:
Just curious as to why `appData.entity_name` would not ever be populated at this stage in the journey? Same goes for the entity number.

Reply:
The prototype included it for the remove journey. It's not currently present for the update journey, no idea why, but since no one requested it, we didn't make any changes to that journey.

![Screenshot 2024-03-27 at 10 27 56](https://github.com/companieshouse/overseas-entities-web/assets/137518530/34017c80-d64b-4fcd-8844-331a45d31ebd)
![Screenshot 2024-03-27 at 10 29 48 (2)](https://github.com/companieshouse/overseas-entities-web/assets/137518530/9cc1614f-ea66-40d5-8e7d-49588e23bb1a)

Reply:
@westy Also, I've made a little change to this block of code.

Reply:
@makhtar-ch Err, thanks I guess.
@mwestacott I guess you might be interested too! 😄